conans/test/functional/toolchains/cmake/cmakedeps/test_cmakedeps_find_module_and_config.py

```diff
@@ -113,8 +113,17 @@ def test_reuse_with_modules_and_config(client):
     client.run("build . -if=install")
 
 
-@pytest.mark.parametrize("find_mode", ["both", "config", "module"])
-def test_transitive_modules_found(find_mode):
+find_modes = [
+    ("both", "both", ""),
+    ("config", "config", ""),
+    ("module", "module", ""),
+    ("both", None, ""),
+    ("both", None, "MODULE")
+]
+
+
+@pytest.mark.parametrize("find_mode_PKGA, find_mode_PKGB, find_mode_consumer", find_modes)
```

Comment:
In my opinion, I prefer to see all the inputs together instead of saving them in a variable (when the list of inputs is small enough), I think it's easier to read the test.

```suggestion
@pytest.mark.parametrize("find_mode_PKGA, find_mode_PKGB, find_mode_consumer", [
    ("both", "both", ""),
    ("config", "config", ""),
    ("module", "module", ""),
    ("both", None, ""),
    ("both", None, "MODULE")
])
```